Traceback (most recent call last):
  File "/home/murakami/LabNote_Murakami/PROTAC-TS/make_feature.py", line 160, in <module>
    main()
  File "/home/murakami/LabNote_Murakami/PROTAC-TS/make_feature.py", line 88, in main
    logger = setup_logger(output_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/murakami/LabNote_Murakami/PROTAC-TS/make_feature.py", line 30, in setup_logger
    fh = logging.FileHandler(filename=os.path.join(savedir, 'run.log'), mode='w')
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/murakami/.pyenv/versions/mambaforge-22.9.0-3/envs/test/lib/python3.11/logging/__init__.py", line 1181, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/home/murakami/.pyenv/versions/mambaforge-22.9.0-3/envs/test/lib/python3.11/logging/__init__.py", line 1213, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/murakami/LabNote_Murakami/PROTAC-TS/result/feature/test/run.log'
start 2025-06-18 16:14:12
end 2025-06-18 16:14:13
